cmd/skopeo/proxy.go

```diff
@@ -116,6 +118,21 @@ type request struct {
 	Args []any `json:"args"`
 }
 
+// Set if we caught EPIPE writing to the client
+const PROXY_ERR_PIPE = "EPIPE"
+
+// Errors that the client believes are retryable (e.g. network flakes)
+const PROXY_ERR_RETRYABLE = "retryable"
+
+// Any other error
+const PROXY_ERR_UNKNOWN = "unknown"
```

Comment:
```go
type proxyErrorCode string
const (
   proxyErrPipe proxyErrorCode = "EPIPE"
   proxyErrRetryable = "retryable"
   …
)
```
and then using a named type in `struct` fields / return values etc. would add a bit of type safety.